src/SonarQube.Client/Api/V7_20/GetIssuesWithComponentRequest.cs

```diff
@@ -0,0 +1,56 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using Newtonsoft.Json;
+
+namespace SonarQube.Client.Api.V7_20
+{
+    internal class GetIssuesWithComponentRequest : GetIssuesRequest
+    {
+        // this class should not be used on it's own
+        private protected GetIssuesWithComponentRequest(){}
+        
+        public virtual string ComponentKey { get; set; }
+    }
+    
```

Comment:
You have some useless whitespace here. I'm not sure if you usually cleanup those or not.
`Trailing Whitespace Visualizer` extensions helps to spot those
```suggestion

        public virtual string ComponentKey { get; set; }
    }

```

Reply:
Usually we ignore trailing spaces, but I will fix it since I'm touching the file anyways